packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/file_scan/ITFileScanTest.java

```diff
@@ -24,7 +24,7 @@ public class ITFileScanTest {
 	final String TESTFILE_PATH = "./src/test/java/cloud/pangeacyber/pangea/testdata/testfile.pdf";
 
 	FileScanClient client;
-	TestEnvironment environment = TestEnvironment.LIVE;
+	TestEnvironment environment = TestEnvironment.STAGING;
```

Comment:
we should not merge SDKs that point to staging i think.

Reply:
No, it was just for testing now, I'll update it once FileScan is on prod